walk.go

```diff
@@ -83,6 +83,10 @@ func (s *PromQLSmith) walkAggregateParam(op parser.ItemType) parser.Expr {
 		return s.Walk(parser.ValueTypeScalar)
 	case parser.COUNT_VALUES:
 		return &parser.StringLiteral{Val: "value"}
+	case parser.LIMITK, parser.LIMIT_RATIO:
+		if s.enableExperimentalPromQL {
```

Comment:
So we pick an aggregation type randomly from enabled aggregate operators. But for now I think default aggregate operators don't include `limitk` and `limit_ratio`.

Reply:
This might go wrong if users enable the feature flag but don't add the operators to enabled aggregate operators